cni/network/network.go

```diff
@@ -608,6 +608,7 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 	endpointIndex := 1
 	for key := range ipamAddResult.interfaceInfo {
 		ifInfo := ipamAddResult.interfaceInfo[key]
+		logger.Info("retrieving interfaceInfo:", zap.Any("ifInfo", ifInfo))
```

Comment:
nit: could we change this to like "Processing interfaceInfo" or similar (we've already retrieved the interface info from the ipam)